include/library/spdm_common_lib.h

```diff
@@ -70,6 +70,8 @@ typedef enum {
     LIBSPDM_DATA_HEARTBEAT_PERIOD,
 
     /* Negotiated result */
+
+    /* LIBSPDM_DATA_LOCAL_USED_CERT_CHAIN_BUFFER is deprecated. Do not use it. Reserved only. */
```

Comment:
You can just delete it, since this will be for the 3.0 release.

Reply:
Thanks. I have deleted it.